Northstar.Client/mod/scripts/vscripts/presence/cl_presence.nut

```diff
@@ -41,6 +48,9 @@ GameStateStruct function DiscordRPC_GenerateGameState( GameStateStruct gs )
         else
             gs.fd_waveNumber = -1 // Tells plugin it's on Wave Break
     }
+	#else
+	gs.fd_waveNumber = -1 // Unecessary for campaign so return -1
+	#endif
```

Comment:
Weird indentation here comes from the fact that the file mixes tabs and spaces :skull:
(Yet another reason we could finally use a proper Squirrel formatter)

Reply:
.editorconfig <3

Reply:
Even then we still need to fix up existing files... :,)